lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -908,14 +913,24 @@ def _test_unicode_data(self, text, mock_request):
         self._set_mock_request_data(mock_request, {
             "closed": False,
         })
-        request = RequestFactory().post("dummy_url", {"body": text})
-        request.user = self.student
-        request.view_name = "create_comment"
-        response = views.create_comment(request, course_id=self.course.id.to_deprecated_string(), thread_id="dummy_thread_id")
+        # We have to get clever here due to Thread's setters and getters.
+        # Patch won't work with it.
+        try:
+            Thread.commentable_id = Mock()
+            request = RequestFactory().post("dummy_url", {"body": text})
+            request.user = self.student
+            request.view_name = "create_comment"
+            response = views.create_comment(
+                request, course_id=self.course.id.to_deprecated_string(), thread_id="dummy_thread_id"
```

Comment:
nitpick: `unicode(self.course.id)` is the convention we've been standardizing on for this.